WooCommerce/Classes/ViewRelated/Products/Media/ProductImageActionHandler.swift

```diff
@@ -180,7 +184,15 @@ final class ProductImageActionHandler {
     private func uploadMediaAssetToSiteMediaLibrary(asset: PHAsset, onCompletion: @escaping (Result<Media, Error>) -> Void) {
         DispatchQueue.main.async { [weak self] in
             guard let self = self else { return }
-            let action = MediaAction.uploadMedia(siteID: self.siteID, productID: self.productID, mediaAsset: asset, onCompletion: onCompletion)
+            let productOrVariationID: Int64 = {
```

Comment:
We could use the computer property to extract the ID here if we decide to add one to `enum ProductOrVariationID`. 😄

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/7179/commits/85e3bd621fd194e90e5bb1b222839bf09c99c7a9